pt/lessons/specifics/ecto.md

```diff
@@ -171,16 +153,11 @@ defmodule ExampleApp.User do
 end
 ```
 
-O esquema que definimos no nosso modelo representa de perto o que especificamos
-na nossa *migration*. Além dos campos para o nosso banco de dados, estamos também a
-incluir dois campos virtuais. Campos virtuais não são armazenados no banco de dados
-mas podem ser úteis em casos de validação. Veremos os campos virtuais em
-acção na secção [Changeset](#changeset).
+O esquema que definimos representa de perto o que especificamos na nossa *migration*. Além dos campos para o nosso banco de dados, estamos também incluindo dois campos virtuais. Campos virtuais não são armazenados no banco de dados mas podem ser úteis em casos de validação. Veremos os campos virtuais em ação na seção [Changesets](#changesets).
 
 ## Consultas
 
-Antes de consultar o nosso repositório, precisamos importar a *API Query*, por
-enquanto precisamos importar apenas `from/2`:
+Antes de poder consultar o nosso repositório, precisamos importar a *API Query*, por enquanto precisamos importar apenas `from/2`:
```

Comment:
I'm in doubt here, regarding the Query API... I think this would translate better: "... precisamos importar a *Query API*, **mas** por enquanto...".

Reply:
I agree, "Query API" fits better.